debug.html

```diff
@@ -5,7 +5,7 @@
 	<title>Maps</title>
 	<meta http-equiv="Content-Type" content="text/html; charset=utf8"/>
 	<meta name="apple-mobile-web-app-capable" content="yes" />
-	<meta name="viewport" content="initial-scale=1.0, minimum-scale=1.0, maximum-scale=2.0, user-scalable=yes, width=device-width, height=device-height" />	
+	<meta name="viewport" content="initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no, width=device-width, height=device-height" />	
```

Comment:
debug.html is not used in the built app. Built apps use index.html with minified JS files. debug.html is only used while developing & testing, not while deploying.